File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanBadRequestException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>HTTP 400. The API rejected the request as malformed.</summary>
    public sealed class PactmanBadRequestException : PactmanApiException
    {
        /// <summary>Initializes the exception from response metadata.</summary>
        /// <param name="message">Human-readable description of the failure.</param>
        /// <param name="init">The metadata the transport collected from the response.</param>
        public PactmanBadRequestException(string message, ApiErrorInit init)
            : base(message, init, ErrorCategory.BadRequest)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanBadRequestException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** HTTP 400. The API rejected the request; see {@link #apiErrors()} for the reasons. */
public class PactmanBadRequestException extends PactmanApiException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message the message to surface.
     * @param init    the response metadata.
     */
    public PactmanBadRequestException(String message, ApiErrorInit init) {
        super(message, init, ErrorCategory.BAD_REQUEST);
    }
}
